src/main/java/org/broadinstitute/hellbender/utils/smithwaterman/SmithWatermanIntelAligner.java

```diff
@@ -0,0 +1,55 @@
+package org.broadinstitute.hellbender.utils.smithwaterman;
+
+
+import com.intel.gkl.smithwaterman.IntelSmithWaterman;
+import org.broadinstitute.gatk.nativebindings.smithwaterman.SWOverhangStrategy;
+import org.broadinstitute.gatk.nativebindings.smithwaterman.SWParameters;
+import org.broadinstitute.gatk.nativebindings.smithwaterman.SWAlignerNativeBinding;
+import org.broadinstitute.hellbender.exceptions.UserException;
+
+
+public final class SmithWatermanIntelAligner implements SmithWatermanAligner {
```

Comment:
this class needs a comment explaining what it is, i.e. it's a version of the SmithWatermanAligner that is optimized for intel hardware